lib/travis/logs/helpers/database.rb

```diff
@@ -16,66 +16,81 @@ module Helpers
+              uri,
+              max_connections: config[:pool],
+              after_connect: ->(c) { after_connect(c) }
+            )
+            conn.loggers << Logger.new($stdout) if config[:sql_logging]
+            conn
           end
 
-          Sequel.default_timezone = :utc
-          conn = Sequel.connect(
-            uri, max_connections: config[:pool], after_connect: after_connect
-          )
-          conn.loggers << Logger.new($stdout) if config[:sql_logging]
-          conn
-        end
+          def uri_from_config(config)
+            host = config[:host] || 'localhost'
+            port = config[:port] || 5432
+            database = config[:database]
+            username = config[:username] || ENV['USER']
 
-        def self.uri_from_config(config)
-          host = config[:host] || 'localhost'
-          port = config[:port] || 5432
-          database = config[:database]
-          username = config[:username] || ENV['USER']
+            params = {
+              user: username,
+              password: config[:password]
+            }
 
-          params = {
-            user: username,
-            password: config[:password]
-          }
+            enc_params = URI.encode_www_form(params)
+            "postgres://#{host}:#{port}/#{database}?#{enc_params}"
+          end
 
-          enc_params = URI.encode_www_form(params)
-          "postgres://#{host}:#{port}/#{database}?#{enc_params}"
-        end
+          def jdbc_uri_from_config(config)
+            host = config[:host] || 'localhost'
+            port = config[:port] || 5432
+            database = config[:database]
+            username = config[:username] || ENV['USER']
 
-        def self.jdbc_uri_from_config(config)
-          host = config[:host] || 'localhost'
-          port = config[:port] || 5432
-          database = config[:database]
-          username = config[:username] || ENV['USER']
+            params = {
+              user: username,
+              password: config[:password]
+            }
 
-          params = {
-            user: username,
-            password: config[:password]
-          }
+            if config[:ssl]
+              params[:ssl] = true
+              params[:sslfactory] = 'org.postgresql.ssl.NonValidatingFactory'
+            end
 
-          if config[:ssl]
-            params[:ssl] = true
-            params[:sslfactory] = 'org.postgresql.ssl.NonValidatingFactory'
+            enc_params = URI.encode_www_form(params)
+            "jdbc:postgresql://#{host}:#{port}/#{database}?#{enc_params}"
           end
 
-          enc_params = URI.encode_www_form(params)
-          "jdbc:postgresql://#{host}:#{port}/#{database}?#{enc_params}"
-        end
+          def connect
+            new.tap(&:connect)
+          end
+
+          def after_connect(conn)
+            command = "SET application_name TO '#{application_name}'"
+            if conn.respond_to?(:exec)
+              return conn.exec(command)
```

Comment:
Are the `return`s here necessary? (And if so, should the last branch have a `return` too?)